lms/envs/common.py

```diff
@@ -234,7 +234,17 @@
     # True.
     'INDIVIDUAL_DUE_DATES': False,
 
-    # Enable Custom Courses for EdX
+    # .. toggle_name: CUSTOM_COURSES_EDX
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: False
+    # .. toggle_description: Set to True to enable Custom Courses for edX, a feature that is more commonly known as
+    #   CCX. Documentation for configuring and using this feature is available at
+    #   https://edx.readthedocs.io/projects/open-edx-ca/en/latest/set_up_course/custom_courses.html
+    # .. toggle_warnings: When set to true, 'lms.djangoapps.ccx.overrides.CustomCoursesForEdxOverrideProvider' will
+    # .. be added to MODULESTORE_FIELD_OVERRIDE_PROVIDERS
+    # .. toggle_use_cases: opt_in, circuit_breaker
+    # .. toggle_creation_date: 2015-04-10
+    # .. toggle_tickets: https://github.com/edx/edx-platform/pull/6636
     'CUSTOM_COURSES_EDX': False,
```

Comment:
Oops, I forgot.  Also, for toggles, we are trying to move these declarations out of common and use a `SettingToggle`.  I don't know if there is a reasonable app to add this in a `toggles.py` file?

There is an active discussion about whether we can also remove the default setting line from common.py here: https://github.com/edx/edx-platform/pull/25332#discussion_r505833366  As soon as this other PR lands, you'll have access to `SettingToggle`, and that should be very soon. :)

Reply:
FYI: `SettingDictToggle` is now both available and in-use. You don't need to update as part of this PR, but for future toggles, this is the way to go.